CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.8.0
+<!-- generated comparing v5.7.0..master -->
+_Apr 4, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add the documentation to save and restore the grid state (#4028) @flaviendelangle
+
+  The different methods to save and restore the data-grid state are now [documented](mui.com/components/data-grid/state/#save-and-restore-the-state)
+
+- ⌚️ Import date-piker components from the lab (#3451) @flaviendelangle
+
+  date-picker components has been added to the MUI-X repo.
+  They are now accessible in their own packages: `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`.
+  For more information, you can read the [blog article](mui.com/blog/lab-pickers-to-mui-x).
+
+- 🐞 Bug and typo fixes
+
+### `@mui/x-data-grid@v5.8.0` / `@mui/x-data-grid-pro@v5.8.0`
+
+#### Changes
+
+- [DataGrid] Add `onProcessRowUpdateError` prop (#4267) @m4theushw
+- [DataGrid] Add generic typing to `GridColDef` and derived interfaces (#4064) @flaviendelangle
```

Comment:
Is this one worth a highlight ?
Some users will not care at all
For others, it might be something very interesting

Reply:
Yes, it's better than "bug fixes" 👍

Reply:
For the example, I would go for the implicit prop typing:

```tsx
const rows: Movie[] = []

return (
  <DataGrid 
    rows={rows}
    columns={[{ 
      // typeof params.row => Movie (R)
      valueGetter: params => params.row.year, 
    }]}
  />
```